### ISSUE TITLE
html link with multiple url parameters conflicts with reference

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#147]
```
Steps to reproduce the problem:
Add an html link in your model using "SyntaxKind.Html" (I am using freemarker 
and docx). The url should contain multiple url parameters connected by "&"
See example in attached "_TestHtmlLink.java" (you will need JUnit4) and 
"HtmlLink.docx".

What is the expected output:
I would like to see a hyperlink in the generated MS-Word not only for  the 
first test but also for the second or third test. Currently MS-Word shows an 
error for the generated document.

Version of the product used: 0.9.8

See also forum entry:
https://groups.google.com/forum/#!topic/xdocreport/AA2A8KJQTVU



```

Original issue reported on code.google.com by `rjedin...@gmail.com` on 17 Sep 2012 at 11:32

Attachments:
- [_TestHtmlLink.java](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-147/comment-0/_TestHtmlLink.java)
- [HtmlLink.docx](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-147/comment-0/HtmlLink.docx)

**Comment by GoogleCodeExporter** [comment]
```
Many thank's for your attached java and docx.
I will study your problem as soon as I will fidn time.

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 17 Sep 2012 at 12:15
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi,

This issue is fixed on Git and will be available on XDocReport 1.0.0. The 
restriction is that you must use "&" token instead of "&" character in your URL 
like "https://www.google.com/search?q=xdocreport&start=10".

See the JUnit testHyperlinkWithURLParameters which tests this issue in the 
http://code.google.com/p/xdocreport/source/browse/document/fr.opensagres.xdocrep
ort.document.docx/src/test/java/fr/opensagres/xdocreport/document/docx/textstyli
ng/DocxDocumentHandlerTestCase.java

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 19 Sep 2012 at 4:15
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified ;) - thanks a lot for fixing the issue so quickly!
```

Original comment by `rjedin...@gmail.com` on 19 Sep 2012 at 3:15
